tests/test.sh

```diff
@@ -598,7 +598,7 @@ function test-build-and-run-csperf-l3-log()
         # No LOC-encoding is in-effect, so no need for the --loc-binary argument.
         ./l3_dump.py                                                                \
                 --log-file /tmp/l3.c-server-test.dat                                \
-                --binary "./build/${Build_mode}/bin/use-cases/svmsg_file_server"    \
+                --binary "tests/build/${Build_mode}/bin/use-cases/svmsg_file_server"    \
```

Comment:
Would be good to consistently stick with the convention I've tried to follow; i.e. `./tests/build/...`

Reply:
You mean start with `./`? Sure, done.